.github/workflows/repo2docker-PR.yaml

```diff
@@ -26,4 +26,4 @@ jobs:
       if: always()
       run: |
         docker images
-        docker run ${{secrets.DOCKER_IMAGE}}:latest conda list --export
+        docker run PR-build:latest conda list --export
```

Comment:
Looks like this change breaks the workflow. See the log output:
```
docker: invalid reference format: repository name must be lowercase.
```

Reply:
whoops!